app/database/item_store.go

```diff
@@ -336,6 +336,10 @@ type activeContestInfo struct {
 	StartTime         time.Time
 }
 
+func (contest *activeContestInfo) IsOver() bool {
+	return contest.Now.After(contest.EndTime) || contest.Now.Equal(contest.EndTime)
```

Comment:
But you still use "After" while my remark was "It is usually easier to reflect about time with "eventA < eventB" instead of "eventA > eventB" so if !activeContest.Now.Before(activeContest.EndTime)". You have a problem with the `Before` function? 😀

Reply:
I would say I just don't have a problem with the `After` function ;) Fixed